tests/factories/reservation_unit.py

```diff
@@ -44,197 +20,81 @@ class ReservationUnitFactory(GenericDjangoModelFactory[ReservationUnit]):
     class Meta:
         model = ReservationUnit
 
+    # IDs
     sku = fuzzy.FuzzyText()
+    uuid = factory.LazyFunction(uuid.uuid4)
+    rank = factory.Sequence(lambda n: n)
+
+    # Strings
     name = fuzzy.FuzzyText()
-    reservation_unit_type = factory.SubFactory("tests.factories.ReservationUnitTypeFactory")
-    origin_hauki_resource = None
+    description = fuzzy.FuzzyText()
+    contact_information = fuzzy.FuzzyText()
+    terms_of_use = fuzzy.FuzzyText()
+    reservation_pending_instructions = ""
+    reservation_confirmed_instructions = ""
+    reservation_cancelled_instructions = ""
+
+    # Integers
+    surface_area = None
+    min_persons = None
+    max_persons = None
+    max_reservations_per_user = None
+    reservations_min_days_before = None
+    reservations_max_days_before = None
+
+    # Datetime
+    reservation_begins = None
+    reservation_ends = None
+    publish_begins = None
+    publish_ends = None
+    max_reservation_duration = None
+    min_reservation_duration = None
+    buffer_time_before = factory.LazyFunction(datetime.timedelta)
+    buffer_time_after = factory.LazyFunction(datetime.timedelta)
+
+    # Booleans
+    is_draft = False
+    is_archived = False
+    require_introduction = False
+    require_reservation_handling = False
+    reservation_block_whole_day = False
+    can_apply_free_of_charge = False
+    allow_reservations_without_opening_hours = False
+
+    # Enums
+    authentication = AuthenticationType.WEAK.value
+    reservation_start_interval = ReservationStartInterval.INTERVAL_15_MINUTES.value
+    reservation_kind = ReservationKind.DIRECT_AND_SEASON.value
+
+    # Forward many-to-one related
```

Comment:
These descriptions would be nice to have in `ReservationUnitType.Meta.fields` also :)

Reply:
Was confused, since there is a model called ReservationUnitType, but you mean the GraphQL object type ReservationUnitType. I'll add them to those as well.

Reply:
Done